tests/virt_autotest/install_package.pm

```diff
@@ -44,14 +50,28 @@ sub install_package {
         $dependency_rpms = 'bridge-utils';
     }
     if ($dependency_repo) {
+        if (check_var('ARCH', 's390x')) {
```

Comment:
" if (check_var('BACKEND', 's390x') && !check_var('S390_DISK', 'ZFCP')) { "
is used in "logs_from_installation_system.pm".   You use (check_var('ARCH', 's390x')).  So I guess your "BACKEND" is "sVirt". Please ignore if there is no contradiction.